suite-native/module-send/src/sendOutputsFormSchema.ts

```diff
@@ -79,7 +79,7 @@ export const sendOutputsFormValidationSchema = yup.object({
                     ),
                 amount: yup
                     .string()
-                    .required()
+                    .required('Amount is required.')
```

Comment:
Intl localization will be part of future PR? If no, please, localize all errors here.

Reply:
Yes, the form schemas have so far no way how to access Intl context since they are defined in the `suite-common` and out of the React tree. 

We will need to inject the translations there somehow. But let's keep this for separate issue. It will need to be handled also for other modules (etc. Passphrase).